app/components/filters/include_new_or_changed.rb

```diff
@@ -0,0 +1,21 @@
+module Filters
+  class IncludeNewOrChanged
```

Comment:
This class is specific to Appointment Facts but has been named as if it can be used for generically.